common/app/views/support/Zurb.scala

```diff
@@ -0,0 +1,31 @@
+package views.support
+
```

Comment:
thanks! it's actually not a million miles away from the michael jackson thing, with simpler tags expanding to <table> based markup